internal/helpers.go

```diff
@@ -0,0 +1,5 @@
+package internal
+
+func ToPtr[T any](v T) *T {
```

Comment:
Nit: `Ptr` would be fine.

Reply:
8d92c92027f44e5d3683a7d8d837dbc0cc29ebf9